index.html

```diff
@@ -152,6 +152,12 @@ <h2>Settings</h2>
       </div>
     </div>
 
-    <script src="script.js"></script>
+    <script src="scripts/script.js"></script>
+    <script src="scripts/export.js"></script>
+    <script src="scripts/backup_edit.js"></script>
+    <script src="scripts/fileLoad.js"></script>
+    <script src="scripts/library.js"></script>
+    <script src="scripts/modal.js"></script>
+    <script src="scripts/settings.js"></script>
```

Comment:
I really don't like this, would rather use [`import()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) in the scripts themselves